### ISSUE TITLE
Resize Item between grids on dragover / drop

### ISSUE THREAD

**Issue by elporfirio** [gridstack/gridstack.js#413]
Following this example:

[http://troolee.github.io/gridstack.js/demo/two.html](http://troolee.github.io/gridstack.js/demo/two.html)

How we can change the size of widget before drop?

[http://imgur.com/yZD1TTq](http://imgur.com/yZD1TTq)

**Comment by saidheerajg** [comment]
+1 I'm looking for the same exact thing. @elporfirio did you find a solution?

**Comment by nelsonad** [comment]
I was able to get this working by setting some node data in my draggable start event.

```
$(".addItem").draggable({
    start: function (event, ui) {
         var el = $(this);
         var nodeData = el.data("_gridstack_node") || {};
         nodeData.width = 4;
         nodeData.height = 3;
         el.data("_gridstack_node", nodeData);
    }
}
```

**Comment by adumesny** [comment]
closing due to solution found and not a lib issue per say

**Comment by adumesny** [comment]
While the above does work, it's generally considered bad UX to have the dragged element and the targeted drop in the grid element be of different size - the dragged should be the same or slightly smaller.
The grid will take the dragged element and figure out the number of rows/column to make it fit by default, unless you override like done above.

**Comment by adumesny** [comment]
real fix is in #1107
